recipes/g2clib/meta.yaml

```diff
@@ -0,0 +1,34 @@
+package:
+    name: g2clib
+    version: 1.5.0
+
+source:
+    fn: g2clib-1.5.0-patch.tar.gz
+    url: http://www.ncl.ucar.edu/Download/files/g2clib-1.5.0-patch.tar.gz
+    md5: f857f49a66918316996eb52399e04120
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - libpng
+        - zlib
+        - jpeg
+        - jasper
+        - gcc
+
+test:
+    commands:
+        - conda inspect linkages g2clib --name _test
```

Comment:
It would be great if these were making an assertion (like: the linkages should include x, y, z, and they should all come from $PREFIX, or something)